Embedded System L1/AVR ATMEGA32/DIO/01-MCAL/01-DIO/DIO_register.h

```diff
@@ -0,0 +1,36 @@
+/*********Author : Khalid Mosaad Zakaria */
+
+/*********Date   :   20/1/2021          */
+/*********Version: 1                     */
+/****************Desc: *******************/
+
+#ifndef _DIO_REGISTER_
+#define _DIO_REGISTER_
+
+#define DDRA *((u8*)0x3A)
+#define DDRC *((u8*)0x34)
+#define DDRB *((u8*)0x37)
+#define DDRD *((u8*)0x31)
+
+#define PORT_A *((u8*)0x3B)
+#define PORT_B *((u8*)0x38)
+#define PORT_C *((u8*)0x35)
+#define PORT_D *((u8*)0x32)
+
+#define PINA *((volatile u8*)0x39)
+#define PINB *((volatile u8*)0x36)
+#define PINC *((volatile u8*)0x33)
+#define PIND *((volatile u8*)0x30)
+
+#endif
+
+
+	
+	
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of headers

![2](https://user-images.githubusercontent.com/44152298/127502084-967ab9b6-4fba-4be7-ba83-0af348f54c31.PNG)

[Sprints_CodeGuideLine.pdf](https://github.com/AhmedEssamMo/Sprints_Wave_9/files/6900596/Sprints_CodeGuideLine.pdf)